Kentico/xperience-by-kentico-sitefinity-migration-tool
# Feature/media url conversion

# Motivation

Which issue does this fix? Fixes #6

## Checklist


## How to test

Use the StartImportContent method to import content. Check urls inside of your Rich Text fields to ensure the direct links have been updated to the permanent urls where applicable.
